cmd/ocm/create/cluster/cmd.go

```diff
@@ -174,6 +176,18 @@ func init() {
 	arguments.SetQuestion(fs, "version", "OpenShift version:")
 	Cmd.RegisterFlagCompletionFunc("version", arguments.MakeCompleteFunc(getVersionOptions))
 
+	fs.StringVar(
+		&args.domainPrefix,
+		"domain-prefix",
+		"",
+		"An optional unique domain prefix of the cluster. This will be used when generating a "+
```

Comment:
Would be nice to know what will be the behaviour  (default) if user does not provide it. WDYT?

Reply:
Sounds good.

Reply:
done